BLSQ/iaso
# IA-2611 Keep old values for change requests

Keep old values for change requests.

Related JIRA tickets : [IA-2611](https://bluesquare.atlassian.net/browse/IA-2611).

## Changes

When a Change Request is approved:

- we keep the old values as well as the new ones
- in order to make it easier to display a diff in the UI

## How to test

1. create a Change Request
2. approve it
3. you can see old values:
    1. in admin `http://localhost:8081/admin/iaso/orgunitchangerequest/` + **ID** + `/change/`
    2. in the DRF UI `/api/orgunits/changes/` + **ID** + `/`

## Print screen / video

![admin](https://github.com/BLSQ/iaso/assets/281139/e38123eb-ea1a-4721-ae89-5d23a5b4a92b)

![api](https://github.com/BLSQ/iaso/assets/281139/124eca5d-087d-42a2-9f60-4f161a1afb75)




[IA-2611]: https://bluesquare.atlassian.net/browse/IA-2611?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
